src/content/en/tools/chrome-user-experience-report/getting-started.md

```diff
@@ -0,0 +1,216 @@
+  <li>Enter your query in the main panel.</li>
+  <li>Execute the query by clicking <b>Run Query</b>.</li>
+</ol>
+
+#### Example queries
+
+Let’s start with a basic query to see if a particular origin is available in the dataset. Paste the query below into the query editor and click Run Query to execute it.
+
+
+```sql
+#standardSQL
+SELECT DISTINCT origin
+FROM `chrome-ux-report.chrome_ux_report.201710`
+WHERE origin LIKE '%://example.com'
+```
+[Run it on BigQuery](https://bigquery.cloud.google.com/savedquery/920398604589:00943c51adc044588842b2e8230d43ce)
+
+Here’s how it works:
+
+<table>
+  <tr>
+    <td><code>#standardSQL</code></td>
+    <td>The first line is a directive that you want to use the <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/">standard SQL</a> syntax.</td>
+  </tr>
+  <tr>
+    <td><code>SELECT</code></td>
+    <td>The SELECT clause expresses the output of the query. In this example, we’re looking for a list of origins which may be HTTP or HTTPS.</td>
+  </tr>
+  <tr>
+    <td><code>FROM</code></td>
+    <td>The FROM clause expresses the project, dataset, and table that you want to query.</td>
+  </tr>
+  <tr>
+    <td><code>WHERE</code></td>
+    <td>Without the WHERE clause on Line 4, the example query would return all of the distinct origins in the dataset. Since we’re only interested in a particular origin, we can use the <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#comparison-operators"><code>LIKE</code></a> operator to compare each origin against a pattern. This query matches all origins that end with “://example.com”. This is to account for a domain that may support both HTTP and HTTPS.</td>
+  </tr>
+</table>
+
+Once we know the origin we would like to examine more closely, we can dive deeper into the user experience data. Let’s write a query to visualize the distribution of the histogram.
+
+```sql
+#standardSQL
+SELECT
+  bin.start,
+  SUM(bin.density) AS density
+FROM (
+  SELECT
+    first_contentful_paint.histogram.bin AS bins
+  FROM
+    `chrome-ux-report.chrome_ux_report.201710`
+  WHERE
+    origin = 'http://example.com')
+CROSS JOIN
+  UNNEST(bins) AS bin
+GROUP BY
+  bin.start
+ORDER BY
+  bin.start
+```
+[Run it on BigQuery](https://bigquery.cloud.google.com/savedquery/920398604589:bb17f33fa95348318e5c685551cbd93d)
+
+The query above produces the data for the histogram by using the <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#sum"><code>SUM</code></a> function to add up the densities for each bin. This query also utilizes a nested query on Line 6 to select the first contentful paint (<a href="/web/fundamentals/glossary#FCP">FCP</a>) histogram’s bins and uses <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#unnest"><code>UNNEST</code></a> on Line 13 to convert the bins from a single row of repeated bins into many rows of a single bin.
+
+<table>
```

Comment:
with only two cols, i recommend you add `class="responsive"`

Reply:
Chatted offline. This class requires consolidating both th into one. Omitting it to keep both headers.